src/app/dynamic-forms/models/cinchy-form.model.ts

```diff
@@ -541,18 +541,24 @@ export class Form {
     let assignmentColumns = new Array<string>();
     let assignmentValues = new Array<string>();
     let attachedFilesInfo = [];
+    let foundLinkedColumn = false;
 
     let paramName: string;
     let paramNumber: number = 0;
     let params: { [key: string]: any } = {};
 
     this.rowId = rowId;
+    const childFormLinkName = this.childFormLinkId?.split("].[")[0]?.replace(/[\[\]]+/g, "");
```

Comment:
Since this appears twice, maybe a utility function would be appropriate. `Form.getChildFormLinkName(childFormLinkId: string): string`?

Reply:
Done, thanks!